test/src/main/scala/scalarules/test/twitter_scrooge/thrift/bare_jar_thrifts/bare_jar_1/BUILD

```diff
@@ -2,8 +2,6 @@ load("//thrift:thrift.bzl", "thrift_library")
 
 thrift_library(
     name = "bare_jar_1",
-    external_jars = [
-        "bare-thrift-1.jar",
-    ],
+    srcs = ["//test/src/main/resources/scalarules/test:Bar.thrift"],
```

Comment:
same as above. Do you know why the second test is needed?

Reply:
now I reference .jar directly
the second test is meant for test_version.sh, apparently every test running against various versions has its copy there